### PR TITLE
fix(pagination): change background position of arrow icon, and global…

### LINKED ISSUE #1119: Docs > Components > Pagination > Working with icons: Fix css
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Left and right arrows seem not to be centered in their squares. They seem to be 1 extra pixel towards right for the left arrow and 1 extra pixel towards left for the right arrow. 

### Reduced test cases

NA

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

v5.1.0

### PR DESCRIPTION
**Pull request by hannahiss**
change background position of arrow icon, and globally all icon-buttons, of 1 pixel so has it is well centered

closes #1119

### CODE REVIEW

**Comment by louismaximepiton** [comment]
:warning: Be careful here, this changes affect : Back-to-top, social buttons, carousel, close-button, quantity selector and also tarteaucitron. I think you could maybe handle the changes directly in `_pagination.scss`.

File: scss/_back-to-top.scss [hunk 1]
```diff
@@ -24,7 +24,12 @@
   position: sticky;
   top: $back-to-top-link-offset-top;
   pointer-events: all;
-  @include button-icon($back-to-top-icon, $size: $back-to-top-icon-width $back-to-top-icon-height, $pseudo: "after");
+  @include button-icon(
```
Comment (louismaximepiton) [inline scss/_back-to-top.scss:27]: Looking back to this, I don't know if this should be in this PR, maybe open a brand new PR with all of your changes !

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
I think back-to-top should be in another PR, the remaining sounds good to me !

File: scss/_back-to-top.scss (hunk 1 above)
Comment (hannahiss) [reply scss/_back-to-top.scss:27]: Ok, but for the new PR I need a new issue ?

**Review by louismaximepiton — APPROVED** [review]
LGTM 🚀

**Review by julien-deramond — APPROVED** [review]

**Comment by julien-deramond** [comment]
LGTM :rocket: 

@hannahiss This issue seems to be there in v4 as well. If you confirm, could you create a PR to fix v4 as well?

**Pull request merged**